src/main.rs

```diff
@@ -178,3 +179,35 @@ async fn main() -> Result<(), Error> {
 
     Ok(())
 }
+
+fn make_message(pull_request: GithubPullRequest, show_pr_age: bool) -> String {
+    let mut message = "".to_string();
+
+    write!(message, "<{}|{}#{}> - {}",
+        pull_request.html_url().replace("https://", ""),
+        pull_request.head().repo().name(),
+        pull_request.number(),
+        pull_request.title()).unwrap();
+
+    if show_pr_age {
+        let age = get_age(Utc::now().with_timezone(&FixedOffset::east_opt(0).unwrap()), *pull_request.created_at()).to_string();
+        write!(message, " - (_{}_)", age).unwrap();
+    }
+
+    write!(message, "\n").unwrap();
+    return message;
```

Comment:
Explicit returns aren't required in rust, I think its a bit more idiomatic to do something like this instead

```suggestion
    message
```

Rust treats this the same way Scala does where the last expression is the return value of a function

Reply:
+1, good spot!

Yeah, it's preferred to use an expression at the end of a function, instead of explicit return.